react-for-atom.js

```diff
@@ -10,21 +10,37 @@ if (typeof atom.__DO_NOT_ACCESS_React_Singleton === 'undefined') {
     // circumvent React Dev Tools console warning
     window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = {};
   }
-  var exportables = {
-    default: function() { return require('react'); },
-    createFragment: function() { return require('react-addons-create-fragment'); },
-    CSSTransitionGroup: function() { return require('react-addons-css-transition-group'); },
-    LinkedStateMixin: function() { return require('react-addons-linked-state-mixin'); },
-    Perf: function() { return require('react-addons-perf'); },
-    PureRenderMixin: function() { return require('react-addons-pure-render-mixin'); },
-    React: function() { return require('react'); },
-    ReactDOM: function() { return require('react-dom'); },
-    ReactDOMServer: function() { return require('react-dom/server'); },
-    shallowCompare: function() { return require('react-addons-shallow-compare'); },
-    TestUtils: function() { return require('react-addons-test-utils'); },
-    TransitionGroup: function() { return require('react-addons-transition-group'); },
-    update: function() { return require('react-addons-update'); },
-  };
+  var exportables = process.env.NODE_ENV !== 'production' ?
+    {
+      default: function() { return require('react'); },
+      createFragment: function() { return require('react-addons-create-fragment'); },
+      CSSTransitionGroup: function() { return require('react-addons-css-transition-group'); },
+      LinkedStateMixin: function() { return require('react-addons-linked-state-mixin'); },
+      Perf: function() { return require('react-addons-perf'); },
+      PureRenderMixin: function() { return require('react-addons-pure-render-mixin'); },
+      React: function() { return require('react'); },
+      ReactDOM: function() { return require('react-dom'); },
+      ReactDOMServer: function() { return require('react-dom/server'); },
+      shallowCompare: function() { return require('react-addons-shallow-compare'); },
+      TestUtils: function() { return require('react-addons-test-utils'); },
+      TransitionGroup: function() { return require('react-addons-transition-group'); },
```

Comment:
Ah yeah, good call. I'll do that in the release when we update to 15.1.0. Thanks!!!